src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/CommonLogger.cs

```diff
@@ -11,12 +13,14 @@ namespace NuGet.Services.Metadata.Catalog.Monitoring
     /// </summary>
     public class CommonLogger : Common.ILogger
     {
-        public CommonLogger(ILogger logger)
+        private static EventId logEvent = new EventId(23847);
+
+        public CommonLogger(Microsoft.Extensions.Logging.ILogger logger)
```

Comment:
It's not too trivial: `Microsoft.Extensions.Logging.ILogger.LogDebug` method and it's clones for other log levels are not actual methods of the `ILogger` class, but extensions. So to test anything that calls those would also mean I would be testing the library implementation of those extensions, which is prone to errors caused by library implementation change.